ftw/linkchecker/linkchecker.py

```diff
@@ -0,0 +1,100 @@
+from multiprocessing import Pool
+import time
+import requests
+import urls_testing
+import os
+import sys
+
+urls = urls_testing.urls
+number_of_processes = len(urls)
+timeout = 1
+
+
+def millis():
+    return int(round(time.time() * 1000))
+
+
+def get_uri_responses(urls):
```

Comment:
For me, only one response is returned 😉 responses -> response

Reply:
Changed that 😄